CMakeLists.txt

```diff
@@ -23,7 +23,6 @@ option(S2N_NO_PQ "Disables all Post Quantum Crypto code. You likely want this
 for older compilers or uncommon platforms." OFF)
 option(S2N_NO_PQ_ASM "Turns off the ASM for PQ Crypto even if it's available for the toolchain.
 You likely want this on older compilers." OFF)
-option(S2N_AWSLC_KYBER_UNSTABLE "Prefer the AWS-LC provided PQ implementation." OFF)
```

Comment:
Hmmm actually, is removing this a breaking change? We might need to just switch the default to ON.

Reply:
i don't think it's a breaking change.

1. it's not used anywhere ([internal search](https://code.amazon.com/search?term=S2N_AWSLC_KYBER_UNSTABLE), [external search](https://github.com/search?q=S2N_AWSLC_KYBER_UNSTABLE&type=code))
2. in my experience, cmake just ignores options it doesn't recognize

Reply:
I tested it and got a warning, but not an error. So I think you're right and we're fine.